test/builder/pipeline.go

```diff
@@ -310,6 +310,17 @@ func PipelineRunServiceAccount(sa string) PipelineRunSpecOp {
 	}
 }
 
+// PipelineRunServiceAccountTask overrides the service account for given Task in PipelineRun.
+func PipelineRunServiceAccountTask(taskName, sa string) PipelineRunSpecOp {
```

Comment:
the functions in this file also have their own tests in https://github.com/tektoncd/pipeline/blob/master/test/builder/pipeline_test.go could you add some coverage for this function there also? :D

Reply:
Done.